### ISSUE TITLE
Master branch is not the final branch (problem in last pull request merge)

### ISSUE THREAD

**Issue by SomnathS09** [androiddevs18/RunningAppYT#4]
There is problem in master branch, please check it's not the final branch. Maybe, because of last pull request; so most recent and updated branch is SettingUpTheBarChart for now

**Comment by Ndrocchietto** [comment]
yes, also if I get the last branch SettingUpTheBarChart is not working properly to me, once I insert the API code of google map and I start the emulator to simulate a running, pressing the button start the map camera does not zoom and the callback latitude and longitude is not recognised by Timber

**Comment by lionheart83** [comment]
Bro, please provide in java languange

**Comment by emirhangulmez** [comment]
> Bro, please provide in java languange

Almost all of projects made by Philipp Lackner are written in kotlin, this is because it is more useful and explains the topics about kotlin on his channel.
